doc/api/tls.md

```diff
@@ -917,7 +917,7 @@ certificate.
    It is returned as a `:` separated hexadecimal string. Example:
    `'2A:7A:C2:DD:...'`.
 * `ext_key_usage` {Array} (Optional) The extended key usage, a set of OIDs.
-* `subjectaltname` {Array} (Optional) An array of names for the subject, an
+* `subjectaltname` {string} (Optional) A string containing concatenated names for the subject, an
```

Comment:
I think the linter is not happy with this long line.

Reply:
fixed